phpBB/phpbb/language/language_file_helper.php

```diff
@@ -45,28 +46,67 @@ public function get_available_languages()
 		// Find available language packages
 		$finder = new Finder();
 		$finder->files()
-			->name('iso.txt')
+			->name('composer.json')
 			->depth('== 1')
 			->followLinks()
 			->in($this->phpbb_root_path . 'language');
 
 		$available_languages = array();
 		foreach ($finder as $file)
 		{
-			$path = $file->getRelativePath();
-			$info = explode("\n", $file->getContents());
+			$json = $file->getContents();
+			$data = sanitizer::decode($json);
 
-			$available_languages[] = array(
-				// Get the name of the directory containing iso.txt
-				'iso' => $path,
-
-				// Recover data from file
-				'name' => trim($info[0]),
-				'local_name' => trim($info[1]),
-				'author' => trim($info[2])
-			);
+			$available_languages[] = $this->get_language_data_from_json($data);
 		}
 
 		return $available_languages;
 	}
+
+	/**
+	 * Collect some data from the composer.json file
+	 *
+	 * @param string $path
+	 * @return array
```

Comment:
Maybe `@throws` here? Also you can alias it if you want to drop the leading backslash from `DomainException`.

Reply:
Added those.